common/amundsen_common/utils/atlas.py

```diff
@@ -36,6 +55,39 @@ class AtlasDashboardTypes:
     execution = 'DashboardExecution'
 
 
+class AtlasRelationshipTypes:
+    group_dashboard = 'DashboardGroup__Dashboard'
+    dashboard_users = 'Dashboard_Users_Owner'
```

Comment:
I think it's dashboard_owners but again - not sure if this is something we would use outside databuilder? If no then it doeesn't make sense to have it here